### PR TITLE
Augur Submission One

### PR DESCRIPTION
**Pull request by sgoggins**
Augur GSoC Submission.

### CODE REVIEW

File: GSoC-Ideas.md [hunk 1]
```diff
@@ -1,5 +1,55 @@
 # Ideas for Google Summer of Code projects
 
+## Idea 5: Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur
+
+### Description
+
+[Augur](http://www.augurlabs.io) is a fully functional prototyping web stack for CHAOSS metrics that leverages cutting edge web technologies including VUE, NodeJS and Python. It provides structured data mined from git repositories using a plugin architecture that incorporate other open source metrics projects like [Facade](http://facade-oss.org/) and [FOSSology](https://www.fossology.org/). Drawing from feedback from the CHAOSS community, Rapid metrics prototyping, human centered design and representing trends over time are central values in Augur's development. Augur has 4 core capabilities:
```
Comment (germonprez) [inline GSoC-Ideas.md]: rapid metrics prototyping (lower case Rapid)

File: GSoC-Ideas.md [hunk 2]
```diff
@@ -1,5 +1,55 @@
...
+1. Open source project comparsisons using either Z-Score or raw data. In the case of two projects of comparable size, scope and scale, percentage comparisons can be useful. In many cases, comparing z-scores of project trends yields a visualization with a more valuable view of how the project’s evolved relative to each other over time.
+2. Comparison of the open source project with itself. How are we doing over time?
+3. Making the open source project’s ecosystem visible. What are the trends in downloads? What projects are dependent on this one? Upon which projects does this project depend?
+4. Supporting Comparisons across open source projects.
+
+We plan to implement Risk metrics and other metrics within the Growth maturituy and decline [CHAOSS metrics and use cases](https://github.com/chaoss/wg-gmd) using Augur, focusing on what we have unearthed as the open source community manager use case. The aims of the project are to:
```
Comment (germonprez) [inline GSoC-Ideas.md]: Growth-Maturity-Decline of a project

File: GSoC-Ideas.md (hunk 1 above)
Comment (sgoggins) [reply GSoC-Ideas.md]: Fixed.

File: GSoC-Ideas.md (hunk 2 above)
Comment (sgoggins) [reply GSoC-Ideas.md]: Added a commit. Fixed.

**Comment by germonprez** [comment]
@sgoggins can you address @GeorgLink comments? This is due really soon.

**Comment by sgoggins** [comment]
I addressed @GeorgLink 's feedback.  I think this is ready to merge.

File: GSoC-Ideas.md [hunk 3]
```diff
@@ -83,3 +83,49 @@ The aims will require extending GrimoireLab functionality to integrate Graal.
...
+
+[ Micro-tasks and place for questions ](#)
+
+[Augur](http://www.augurlabs.io) is a fully functional prototyping web stack for CHAOSS metrics that leverages cutting edge web technologies including VUE, NodeJS and Python. It provides structured data mined from git repositories using a plugin architecture that incorporate other open source metrics projects like [Facade](http://facade-oss.org/) and [FOSSology](https://www.fossology.org/). 
+
+In this GSoC project we plan to implement Risk metrics and other metrics within the Growth-Maturity-Decline [CHAOSS metrics and use cases](https://github.com/chaoss/wg-gmd) using Augur, focusing on what we have unearthed as the open source community manager use case. The aims of the project are to: 
+1. Construct core open source software metrics defined by CHAOSS and
+2. Assemble filtered, comparative collections of those metrics in ways that community managers want to consume. 
+
+The aims of the project are as follows:
```
Comment (GeorgLink) [inline GSoC-Ideas.md]: Lines 94 - 101 have two lists of aims, can you combine them?

File: GSoC-Ideas.md [hunk 4]
```diff
@@ -83,3 +83,49 @@ The aims will require extending GrimoireLab functionality to integrate Graal.
   * _Requirements:_ Python programming. Interest in software analytics. Willingness to understand GrimoireLab internals.
   * _Recommended:_ Experience with ElasticSearch and Kibana would be convenient, but can be learned during the project.
   * _Mentors:_ Jesus M. Gonzalez-Barahona, Valerio Cosentino
+  
+## Idea 5: Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur
+
+[ Micro-tasks and place for questions ](#)
```
Comment (GeorgLink) [inline GSoC-Ideas.md]: Line 89 needs the link to the issue with the micro-tasks

Comment (GeorgLink) [reply GSoC-Ideas.md]: I found the link to the micro-tasks and added it.

**Pull request merged**